OneBlink.SDK/OrganisationsClient.cs

```diff
@@ -49,8 +49,11 @@ public async Task<string> UploadAsset(Stream assetDataStream, string contentType
                 Key = assetUploadCredentialsResponse.s3.key,
                 InputStream = assetDataStream,
                 ContentType = contentType,
-                CannedACL = S3CannedACL.PublicRead
+                CannedACL = S3CannedACL.PublicRead,
+                ServerSideEncryptionMethod = ServerSideEncryptionMethod.AES256,
             };
+            request.Headers.ExpiresUtc = new DateTime().AddYears(1).ToUniversalTime(); // Max 1 year
+            request.Headers.CacheControl = "max-age=31536000"; // Max 1 year(365 days)
```

Comment:
It does not look like we are adding a unique key here to s3, prob don't want to cache these. Or we could add a unique key to the path. I expect thats easy enough in C#